js/languages/fr.js

```diff
@@ -0,0 +1,326 @@
+    'staff-error': "; impossible d'obtenir les données du personnel; peut-être que vous n'êtes pas connecté à Internet?",
+    title: 'Titre:',
+    department: 'Département:',
+    email: 'Email:',
+    phone: 'Téléphone:',
+    website: 'Site Internet:',
+    basement: 'Sous-sol:',
+    'oc-basement': "Sous-sol d'OC",
+    'sophomore-club': 'Club Sans Seniors',
+    'soph-desc': "C'est un club sans seniors!",
+    'soph-day': 'Jeudi',
+    'soph-time': 'Flex',
+    'soph-room': "N'importe quel salle",
+    'soph-prez': 'Tara Firenzi',
+    'soph-teacher': 'Courtney Carlomagno',
+    'soph-email': 'ccarlomagno@pausd.org',
+    'club-error': "; impossible d'obtenir les données du club; peut-être que vous n'êtes pas connecté à Internet?",
+    day: 'Jour de réunion:',
+    time: 'Heure de réunion:',
+    location: 'Emplacement:',
+    desc: 'Description:',
+    presidents: 'Président(s):',
+    advisors: 'Conseiller(s) Professeur:',
+    'teacher-email': 'Email de Prof:',
+    donation: 'Don suggéré:',
+    'before-alt-msg': "Ceci est un horaire alternatif. L'école dit, \"",
+    'after-alt-msg': '"',
+    appname: 'Unofficial Gunn Web App (UGWA)',
+    'add-to-list': 'Ajouter à mes clubs',
+    'remove-from-list': 'Supprimer de mes clubs',
+    'lunch-clubs': 'Clubs',
+    summer: 'Profitez de votre été!',
+    'image-url': "URL de l'image",
+    cannot: "Un problème est survenu lors de la récupération de l'image.",
+    'add-asgn': 'Ajouter un devoir',
+    asgn: 'Devoirs à venir',
+    overdue: 'en retard',
+    doneify: 'Marque comme terminé',
+    undoneify: 'Démarquer comme terminé',
+    'asgn-cat-homework': 'devoirs',
+    'asgn-cat-preparation': 'préparation',
+    'asgn-cat-worksheet': 'feuille de travail',
+    'asgn-cat-reading': 'lecture',
+    'asgn-cat-quiz': 'quiz',
+    'asgn-cat-test': 'test',
+    'asgn-cat-exam': 'examen',
+    'asgn-cat-presentation': 'présentation',
+    'asgn-cat-materials': 'matériels',
+    'asgn-cat-lab': 'laboratoire',
+    'asgn-cat-other': 'autre',
+    'import-warning': 'Cela remplacera définitivement vos paramètres existants. Veux-tu çà?',
+    'import-problem': 'Il y avait un problème.',
+    'export-file-name': "les-paramètres-d'un-étudiant-intellectuel-gunn.json",
+    'assync-loading': 'Chargement des devoirs...',
+    'assync-loaded': 'Devoirs chargées.',
+    'assync-loading-problem': 'Un problème est survenu lors du chargement des devoirs.',
+    'assync-saving': 'Enregistrer les modifications...',
+    'assync-saved': 'Changements sauvegardés.',
+    'assync-saving-problem': "Un problème est survenu lors de l'enregistrement des modifications.",
+    p0: 'Zéro période',
+    'dead-club': "Ce club n'existe plus à Gunn.",
+    'psa-date': 'Créé sur {D}.',
+    'psa-error': 'Impossible de charger le PSA:'
+  },
+  placeholders: {
+    clubs: 'Recherche clubs',
+    staff: 'Recherche de personnel',
+    'send-msg': 'Envoyer un message',
+    assignment: 'Devoir',

```

Comment:
An assignment isn't necessarily homework.

Reply:
What would you put instead?